Session/Onboarding/LandingScreen.swift

```diff
@@ -0,0 +1,259 @@
+            info: ConfirmationModal.Info(
+                title: "urlOpen".localized(),
+                body: .text("urlOpenBrowswer".localized()),
+                confirmTitle: "terms_of_service".localized(),
+                confirmAccessibility: Accessibility(
+                    identifier: "Terms of service button",
+                    label: "Terms of service button"
+                ),
+                confirmStyle: .textPrimary,
+                cancelTitle: "privacy_policy".localized(),
+                cancelAccessibility: Accessibility(
+                    identifier: "Privacy policy button",
+                    label: "Privacy policy button"
+                ),
+                cancelStyle: .textPrimary,
+                onConfirm: { _ in
+                    if let url: URL = URL(string: "https://getsession.org/terms-of-service") {
+                        UIApplication.shared.open(url)
+                    }
+                },
+                onCancel: { _ in
+                    if let url: URL = URL(string: "https://getsession.org/privacy-policy") {
+                        UIApplication.shared.open(url)
+                    }
+                }
+            )
+        )
+        self.host.controller?.present(modal, animated: true)
+    }
+}
+
+struct ChatBubble: View {
+    let text: String
+    let outgoing: Bool
+    
+    var body: some View {
+        let backgroundColor: Color? = ThemeManager.currentTheme.colorSwiftUI(for: (outgoing ? .messageBubble_outgoingBackground : .messageBubble_incomingBackground))
+        Text(text)
+            .foregroundColor(themeColor: (outgoing ? .messageBubble_outgoingText : .messageBubble_incomingText))
+            .font(.system(size: 16))
+            .padding(.all, 12)
+            .background(backgroundColor)
+            .cornerRadius(13)
+            .frame(
+                maxWidth: 230,
+                alignment: (outgoing ? .trailing : .leading)
+            )
+    }
+}
+
+struct FakeChat: View {
+    @State var numberOfBubblesShown: Int = 0
+    
+    let chatBubbles: [ChatBubble] = [
+        ChatBubble(text: "onboardingBubbleWelcomeToSession".localized() + " 👋", outgoing: false),
+        ChatBubble(text: "onboardingBubbleSessionIsEngineered".localized(), outgoing: true),
+        ChatBubble(text: "onboardingBubbleNoPhoneNumber".localized(), outgoing: false),
+        ChatBubble(text: "onboardingBubbleCreatingAnAccountIsEasy".localized() + " 👇", outgoing: true),
+    ]
+    
+    var body: some View {
+        VStack(
+            alignment: .leading,
+            spacing: 18
+        ) {
+            ForEach(
+                0...(chatBubbles.count - 1),
+                id: \.self
+            ) { index in
+                let chatBubble: ChatBubble = chatBubbles[index]
+                let bubble = chatBubble
+                    .frame(
+                        maxWidth: .infinity,
+                        alignment: chatBubble.outgoing ? .trailing : .leading
```

Comment:
Just check, we didn't put the incoming/outgoing bubbles on the other side. But when setting the system language to be a RTL languages, everything will be mirrored, so it will end up on the incoming messages on the right (trailing) and the outgoing messages on the left(leading) 
![image](https://github.com/oxen-io/session-ios/assets/8543570/9876a50d-b74b-4790-9a45-cae4fbe4939c)

Reply:
Oh right because we are using `leading`/`trailing` instead of `left`/`right` 🤦 

Thanks for confirming 🙏